index.html

```diff
@@ -98,7 +98,7 @@ <h3 class="featured-content-title">Multi-Post Stories</h3>
       </div>
 
       <div class="second-project"> 
```

Comment:
### Your buttons
![button-contrast](https://user-images.githubusercontent.com/93039359/178567751-cd2ddf75-89c4-43ec-9e52-f085b30b3b7d.png)


- [ ] Please adjust the contrast of your see project button. It should be 4.5:1 :+1: